TaskModules/powershell/TaskModuleIISManageUtility/AppCmdOnTargetMachines.ps1

```diff
@@ -43,7 +43,7 @@ function Test-WebsiteExist
     $command = "`"$appCmdPath`" $appCmdArgs"
     Write-Verbose "Checking website exists. Running command : $command"
 
-    $website = Invoke-VstsTool -Filename $appCmdPath -Arguments $appCmdArgs
+    $website = Invoke-VstsTool -Filename $appCmdPath -Arguments $appCmdArgs -Encoding ([System.Text.Encoding]::GetEncoding(1252))

```

Comment:
@AThomsen instead of making this ::GetEncoding(1252), can this instead be [System.Text.Encoding::DefaultEncoding

Reply:
It works for me by adding 1252. Haven't tried anything else.

Microsoft is really the one to ask about this as the correct solution requires insight in how "appcmd" (closed source, MS) is put together.